### ISSUE TITLE
"for in" and "for of" loops break React Native Android code

### ISSUE THREAD

**Issue by joncursi** [nicolaslopezj/meteor-apollo-accounts#73]
This repository contains several instances of `for in` and `for of` which breaks code execution on React Native Android apps.

Example: https://github.com/orionsoft/meteor-apollo-accounts/blob/3ce4f66b6cf294a96bb9d4c20d4ee43ecbb0a1be/client/src/store.js#L40

Attempting to login with password on android causes this error:

```
09-11 18:09:31.440  3541  3584 I ReactNativeJS: { [TypeError: undefined is not a function (evaluating 'onChangeCallbacks[typeof Symbol === 'function' ? Symbol.iterator : '@@iterator']()')]
```

See a similar issue logged in the `css-to-react-native` repo: https://github.com/styled-components/css-to-react-native/issues/31 - the problem was solved by eliminating these special loops in favor of more standard ES5 code.

**Comment by joncursi** [comment]
More explanation here: https://github.com/facebook/react-native/issues/4676#issuecomment-163399041

**Comment by acomito** [comment]
Is this what is breaking the logout on android? @joncursi

**Comment by joncursi** [comment]
@acomito Yeah, this library doesn't really work on React Native android devices due to Android not natively supporting `Symbol`. You can polyfill it with `core-js` though:

```js
// index.android.js
import 'core-js/es6/symbol';
import 'core-js/fn/symbol/iterator';
```

But beware, doing so seems to [cause other issues](https://github.com/facebook/react-native/issues/15497#issuecomment-332623364).

**Comment by ujwal-setlur** [comment]
@joncursi, I used the proposed solution at:

https://github.com/facebook/react-native/issues/4676#issuecomment-352815780

**Comment by ujwal-setlur** [comment]
This will fail on react-native 0.56.0. It will trigger

https://github.com/facebook/react-native/issues/18542
